### PR TITLE
Enumerate node packages

### PR DESCRIPTION
**Pull request by ddavidebor**
Simply prints a few node versions for ease of debug and record keeping

### CODE REVIEW

**Comment by ddavidebor** [comment]
Fixes #134

**Comment by revant** [comment]
`node`, `npm` and `yarn` might not work in frappe-nginx images. nginx:latest layer does not have node/npm/yarn installed. Does not require nodejs in nginx:latest image

for socketio images:

I prefer not to add layers to docker image by additional `RUN` commands. Can the command be executed in existing layer?

**Comment by ddavidebor** [comment]
Will check and fix accordingly within a couple of days

File: build/frappe-socketio/Dockerfile [hunk 1]
```diff
@@ -30,5 +30,8 @@ RUN ln -s /usr/local/bin/docker-entrypoint.sh / # backwards compat
 
 WORKDIR /home/frappe/frappe-bench/sites
 
+RUN node --version
```
[the remark this answers is not in the document]
Comment (ddavidebor) [reply build/frappe-socketio/Dockerfile]: sure, I'll do that

File: build/frappe-nginx/Dockerfile [hunk 2]
```diff
@@ -31,6 +31,10 @@ COPY build/frappe-nginx/docker-entrypoint.sh /
 
 RUN apt-get update && apt-get install -y rsync && apt-get clean
 
+RUN node --version
```
[the remark this answers is not in the document]
Comment (ddavidebor) [reply build/frappe-nginx/Dockerfile]: I will check and fix it

[the remark this answers is not in the document]
Comment (ddavidebor) [reply build/frappe-nginx/Dockerfile]: fixed

File: build/frappe-socketio/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (ddavidebor) [reply build/frappe-socketio/Dockerfile]: done

File: build/frappe-nginx/Dockerfile [hunk 3]
```diff
@@ -23,6 +23,10 @@ RUN git clone --depth 1 https://github.com/frappe/bench /tmp/bench \
 RUN cp -R /home/frappe/frappe-bench/apps/frappe/frappe/public/* /home/frappe/frappe-bench/sites/assets/frappe \
     && cp -R /home/frappe/frappe-bench/apps/frappe/node_modules /home/frappe/frappe-bench/sites/assets/frappe/
 
+RUN node --version
```
Comment (revant) [inline build/frappe-nginx/Dockerfile]: Can this be part of previous layer?

```Dockerfile
...
&& cp -R /home/frappe/frappe-bench/apps/frappe/node_modules /home/frappe/frappe-bench/sites/assets/frappe/ \
&& node --version \
&& npm --version \
&& yarn --version
```

File: build/frappe-nginx/v11.Dockerfile [hunk 4]
```diff
@@ -23,6 +23,10 @@ RUN git clone --depth 1 https://github.com/frappe/bench /tmp/bench \
 RUN cp -R /home/frappe/frappe-bench/apps/frappe/frappe/public/* /home/frappe/frappe-bench/sites/assets/frappe \
     && cp -R /home/frappe/frappe-bench/apps/frappe/node_modules /home/frappe/frappe-bench/sites/assets/frappe/
 
+RUN node --version
```
Comment (revant) [inline build/frappe-nginx/v11.Dockerfile]: Can this be part of previous layer?

```Dockerfile
...
&& cp -R /home/frappe/frappe-bench/apps/frappe/node_modules /home/frappe/frappe-bench/sites/assets/frappe/ \
&& node --version \
&& npm --version \
&& yarn --version
```

File: build/frappe-nginx/v12.Dockerfile [hunk 5]
```diff
@@ -23,6 +23,10 @@ RUN git clone --depth 1 https://github.com/frappe/bench /tmp/bench \
 RUN cp -R /home/frappe/frappe-bench/apps/frappe/frappe/public/* /home/frappe/frappe-bench/sites/assets/frappe \
     && cp -R /home/frappe/frappe-bench/apps/frappe/node_modules /home/frappe/frappe-bench/sites/assets/frappe/
 
+RUN node --version
```
Comment (revant) [inline build/frappe-nginx/v12.Dockerfile]: Can this be part of previous layer?

```Dockerfile
...
&& cp -R /home/frappe/frappe-bench/apps/frappe/node_modules /home/frappe/frappe-bench/sites/assets/frappe/ \
&& node --version \
&& npm --version \
&& yarn --version
```

Comment (ddavidebor) [reply build/frappe-nginx/v12.Dockerfile]: Done

File: build/frappe-nginx/v11.Dockerfile (hunk 4 above)
Comment (ddavidebor) [reply build/frappe-nginx/v11.Dockerfile]: Done

File: build/frappe-nginx/Dockerfile (hunk 3 above)
Comment (ddavidebor) [reply build/frappe-nginx/Dockerfile]: Done

**Comment by revant** [comment]
https://github.com/frappe/frappe_docker/issues/133

**Pull request merged**